resilience4j-framework-common/src/main/java/io/github/resilience4j/common/circuitbreaker/monitoring/endpoint/CircuitBreakerEventDTOFactory.java

```diff
@@ -54,6 +54,10 @@ public static CircuitBreakerEventDTO createCircuitBreakerEventDTO(CircuitBreaker
                 CircuitBreakerOnCallNotPermittedEvent onCallNotPermittedEvent = (CircuitBreakerOnCallNotPermittedEvent) event;
                 return newCircuitBreakerEventDTOBuilder(onCallNotPermittedEvent)
                     .build();
+            case FAILURE_RATE_EXCEEDED:
```

Comment:
I think we need to add `SLOW_CALL_RATE_EXCEEDED` type too. 
What do you think @RobWin ?

Reply:
Yes

Reply:
@rusyasoft Please add `SLOW_CALL_RATE_EXCEEDED ` too.

Reply:
@dlsrb6342  added SLOW_CALL_RATE_EXCEEDED case